build.sbt

```diff
@@ -476,13 +470,21 @@ lazy val scaldingHRaven = module("hraven").settings(
       exclude("com.twitter.common", "application-module-log")
       exclude("com.twitter.common", "application-module-stats")
       exclude("com.twitter.common", "args")
-      exclude("com.twitter.common", "application"),
+      exclude("com.twitter.common", "application")
+      // Adding this dependencies explicitly because otherwise they are broken
+      exclude("com.twitter", "util-registry_2.10")
+      exclude("com.twitter", "util-core_2.10")
+      exclude("com.twitter", "util-jvm_2.10"),
     "org.apache.hbase" % "hbase" % hbaseVersion,
     "org.apache.hbase" % "hbase-client" % hbaseVersion % "provided",
     "org.apache.hbase" % "hbase-common" % hbaseVersion % "provided",
     "org.apache.hbase" % "hbase-server" % hbaseVersion % "provided",
     "org.slf4j" % "slf4j-api" % slf4jVersion,
-    "org.apache.hadoop" % "hadoop-client" % hadoopVersion % "provided"
+    "org.apache.hadoop" % "hadoop-client" % hadoopVersion % "provided",
+
+    "com.twitter" % "util-registry_2.10" % "6.34.0",
+    "com.twitter" % "util-core_2.10" % "6.34.0",
+    "com.twitter" % "util-jvm_2.10" % "6.34.0", 
```

Comment:
Looking at the rest of the code here I think the pattern would be to capture the version in a variable, like hbaseVersion and hadoopVersion.

Why are these Scala 2.10?

Reply:
Because that's what hraven depends on... See more context in comment above.

Reply:
Removed it for now, it compiles with this excludes successfully, will be up to users of this module to add this deps